networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/GossipForkSubscriptions.java

```diff
@@ -40,4 +40,6 @@
   void subscribeToSyncCommitteeSubnet(int subnetId);
 
   void unsubscribeFromSyncCommitteeSubnet(int subnetId);
+
+  int getMessageMaxSize();
```

Comment:
Does this need to be on the public interface or could it just be a protected method in `GossipForkSubscriptionsPhase0`?

Reply:
yes, true.